sdk/keyvault/azure-keyvault-administration/setup.py

```diff
@@ -79,7 +80,7 @@
             "azure.keyvault",
         ]
     ),
-    install_requires=["azure-common~=1.1", "azure-core<2.0.0,>=1.7.0", "msrest>=0.6.0"],
+    install_requires=["azure-common~=1.1", "azure-core<2.0.0,>=1.7.0", "msrest>=0.6.21"],
```

Comment:
This needs an update too:
https://github.com/Azure/azure-sdk-for-python/blob/e08c2308d682da6f2497c8119824d53af2d1d457/shared_requirements.txt#L129

Reply:
Okay, glad this is a simple fix!